crates/rbuilder/src/bin/debug-fake-order-sim.rs

```diff
@@ -23,6 +25,7 @@ pub async fn main() -> eyre::Result<()> {
 
     let test_context = TestChainState::new(BlockArgs::default().number(11))?;
 
+    // Create simulation core
```

Comment:
why is this comment here? weird above the tokio cancellation token instantiation, move one (or more) line(s) below?